manifests/02_csi_driver_operators_namespace.yaml

```diff
@@ -10,3 +10,6 @@ metadata:
     workload.openshift.io/allowed: "management"
   labels:
     openshift.io/cluster-monitoring: "true"
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/warn: privileged
```

Comment:
For historical reasons, the Manila CSI driver is deployed to a different namespace than the other CSI drivers (_assets/csidriveroperators/manila/01_namespace.yaml_). That driver also has a component that needs to be privileged. 

Could you add these labels to that namespace as well? Note that that NS is created by this operator, not CVO.

Reply:
@bertinatto added the labels, thank you!